### PR TITLE
Hardware milestone

### PR DESCRIPTION
**Pull request by AlkhatibHoda**
### Pull Request Overview

This pull request adds hardware schematic and a photo with the breadboard. 

### Testing Strategy

This pull request was tested by Alkhatib Hoda.

### TODO or Help Wanted

This pull request needs feedback.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Alkhatib Hoda <hudaalkhatib7@gmail.com>

### CODE REVIEW

**Comment by AlkhatibHoda** [comment]
@alexandruradovici

File: docs/project/2024/alkhatib.hoda/Hardware
Comment (alexandruradovici) [inline docs/project/2024/alkhatib.hoda/Hardware:1]: What is this file?

File: docs/project/2024/alkhatib.hoda/Schematic
Comment (alexandruradovici) [inline docs/project/2024/alkhatib.hoda/Schematic:1]: What is this file?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please fix the build errors.

File: docs/project/2024/alkhatib.hoda/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+| [Passive Buzzer](https://ardushop.ro/ro/electronica/194-buzzer.html?search_query=buzzer&results=16)                                                                                       | Buzzer              | 3,97 RON                                                                                                                        |
+| [Joystick mode](https://ardushop.ro/ro/electronica/127-modul-joystick.html?search_query=joystick&results=4)                                                                                        | control the game    | 3,60 RON                                                                                                                                                                                                                                                        |
+| [A LED]()                                                       | turns on when starting | 0 RON-I borrowed it                                                                                                                                                                                                                                                       |
```
Comment (Gabi10017) [inline docs/project/2024/alkhatib.hoda/index.md:94]: Add link inside "()" brackets or remove "[]" and "()"

File: docs/project/2024/alkhatib.hoda/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+| [A resistor]()                                            | current-limiting resistor| 0 RON-I borrowed it                                                                                                                                                                                                                                                        |
```
Comment (Gabi10017) [inline docs/project/2024/alkhatib.hoda/index.md:95]: Add link inside "()" brackets or remove "[]" and "()"

File: docs/project/2024/alkhatib.hoda/index.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+My project is a recreation of the popular game 'Snake,' displayed on an 8×8 LED Matrix module using a Raspberry Pi.My aim is to create something that appears simple but holds many memories of my childhood and provides enjoyable entertainment.
+
+## Motivation
+
+Creating a Snake game with Rust on a Raspberry Pi and LED matrix blends nostalgia with technical challenge, fostering both creativity and learning.
+
+## Architecture
+
+![Architecture.jpg](https://github.com/AlkhatibHoda/upb-fils-ma.github.io/assets/163418706/9adce84e-7836-41bc-a30b-dc0bfab75834)
```
Comment (Gabi10017) [inline docs/project/2024/alkhatib.hoda/index.md:22]: Add "Architecture.jpg" to the comitted files and update the link accordingly (e.g (./Architecture.jpg))

**Review by Gabi10017 — CHANGES_REQUESTED** [review]
Please address the review's comments in next commits, otherwise the build will fail.

File: docs/project/2024/alkhatib.hoda/index.md (hunk 3 above)
Comment (alexandruradovici) [reply docs/project/2024/alkhatib.hoda/index.md]: Use the local file.

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**